verticapy/connection/read.py

```diff
@@ -14,7 +14,9 @@
 See the  License for the specific  language governing
 permissions and limitations under the License.
 """
+
 import os
+import ast
```

Comment:
what is ast, is it standard?
Use A-Z order please

Reply:
https://docs.python.org/3/library/ast.html

This basically helps to keep the dictionary as a dictionary instead of converting it into string when importing. This was creating issues for oauth_refresh_token. 

And changed order.